config/README.md

```diff
@@ -2,9 +2,19 @@
 
 This is a central place for Kubernetes-project specific configs for other tools in this repo.
 
-### UNDER MIGRATION
-
-Directory structure:
+## Directory structure:
 
 [jobs/](./jobs) : job configs for Kubernetes Prow deployment, potentially testgrid configs as well  
 [tests](./tests) : validation tests for the configs
+
+## Adding new jobs to Prow:
+
+1. Find or create a Org/Repo subdir under config/jobs.
```

Comment:
Find or create an org/repo directory under `config/jobs`, eg: `config/jobs/kubernetes-sigs/kustomize` for jobs related to https://github.com/kubernetes-sigs/kustomize

Reply:
I like that I can just copy-paste your line :-)